tests/src/App/Metadata.cpp

```diff
@@ -147,19 +173,14 @@ TEST(VersionTest, VersionOperatorComparison){
 class MetadataTest : public ::testing::Test {
 protected:
     void SetUp() override {
-        try {
-            xercesc_3_2::XMLPlatformUtils::Initialize();
-        }
-        catch (const xercesc_3_2::XMLException& toCatch) {
-            // Some kind of Google Test error condition?
-        }
+        xercesc_3_2::XMLPlatformUtils::Initialize();
     }
     void TearDown() override {
         xercesc_3_2::XMLPlatformUtils::Terminate();
     }
     std::string GivenSimpleMetadataXMLString()
     {
-        std::stringstream stream;
+        std::ostringstream stream;
```

Comment:
@wwmayer do you have any idea why the linter is complaining about this line? There should be no need to do anything to initialize an `ostringstream`, as far as I am aware.

Reply:
No idea what it is complaining about. Sometimes the tool just tells you nonsense. But what you could try is to write: `std::ostringstream stream{};`